processor/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'java-library'
+apply plugin: 'kotlin'
+apply plugin: 'kotlin-kapt'
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    kapt 'com.google.auto.service:auto-service:1.0-rc2'
+    implementation 'com.google.auto.service:auto-service:1.0-rc2'
+    implementation project(':anontation')
+    implementation 'com.squareup:kotlinpoet:1.2.0'
+}
+
+sourceCompatibility = "7"
```

Comment:
why not 8 as @gregriggins36 mentioned?

Reply:
Yea I thought about it, I don't know if its useful for the sourceCompatibility to be the same as the JVM Target.. Wouldn't it be safer for it to be compatible with a lesser version?